crates/red_knot_python_semantic/resources/mdtest/shadowing/function.md

```diff
@@ -25,3 +25,27 @@ def f(): ...
 
 f: int = 1
 ```
+
+## Explicit shadowing involving `def` statements
+
+```py path=a.py
+from typing_extensions import reveal_type
+
```

Comment:
We support `reveal_type` as a built-in, so it's not necessary to import it, and we usually don't in tests.
```suggestion
```